VeganVirus/VeganVirus/ClipBoardAction.cpp

```diff
@@ -0,0 +1,59 @@
+#include "ClipBoardAction.h"
+
+
+ClipBoardAction::ClipBoardAction(double req) : Action(req, CLIP_BOARD_ICON_PATH)
+{
+}
+
+void ClipBoardAction::start()
+{
+	this->activeFlag = true;
+}
+
+void ClipBoardAction::update(double dt)
+{
+	if (!this->activeFlag)
+	{
+		return;
+	}
+	std::string data = this->ClipBoardRead();
+	if (data.find("meat") != std::string::npos)
+	{
+		this->clipBoardWrite("COWS ARE SMARTER THAN DOGS TEL_AVIV");
```

Comment:
First, you should update the progress in here. (lower it by maybe 1-2%?)
Also, don't you want to replace other words? And maybe replace only the words, like in the keyboard action?
I'll let you decide what to do, but I don't think there should be only one word here (you can also decide to change it later).

Reply:
I will decide later about the sentence because I don't find it critical now

Reply:
please explain what you meant by lower it by maybe 1-2%?